src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,23 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The method name 'getStatistic' could be more informative. Consider renaming it to reflect what kind of statistics it is getting or what it does with the statistics.